### ISSUE TITLE
Define possible technical mitigations for the binding app

### ISSUE THREAD

**Issue by alevindecanini** [department-of-veterans-affairs/va.gov-team#57180]
## Issue Description
_The Deferred Proofing pathway has a dependency on the IAM team to build the binding app. Work on the binding app is currently blocked by a number of issues, including: policy, technical, and security concerns. In order to unblock this work, we want to define technical mitigation tactics which can address some or all of these concerns._

---
## Tasks
- [x] Consider and document alternate solutions for deferred proofing which either do not have dependencies on IAM team or reduce the dependencies on IAM team.

## Acceptance Criteria
- [ ] _Engineering & Product teams agree that all items on the list of mitigation tactics are technically feasible_
- [ ] _Engineering & Product teams agree that all items on the list of mitigation tactics address the concerns which created the block_
- [ ] _Any UX impacts have been discussed with Design & Product_
- [ ] _Engineering & Product teams agree that we are ready to present mitigation tactics to our Product Owner_

**Comment by alevindecanini** [comment]
@sdickson-gt @elizabethkochBT can you please fill in the list of tasks for this ticket?

**Comment by sdickson-gt** [comment]
There does not appear to be a technical blockers.  Once the security and policy risks are mitigated, it could be developed by IAM team.  There are alternatives to reduce dependencies on IAM team which could expedite timeline of delivery with associated pros/cons.

**Comment by sdickson-gt** [comment]
@elizabethkochBT @alevindecanini do you want to discuss the acceptance criteria on this so we can close this out?

**Comment by sdickson-gt** [comment]
Core team is going to provide follow-up comments from meeting last week

**Comment by alevindecanini** [comment]
John and Tom are discussing, will meet soon to review
